docs/writing-tests/idlharness.md

```diff
@@ -129,6 +104,18 @@ and outside callers should not use it.
   `document.head` would only be tested for `HTMLHeadElement`, and so on for
   further objects.
 
-### `test()`
-  Run all tests.  This should be called after you've called all other
-  methods to add IDLs and objects.
+### `add_idls(idl_string)`
+  Parses _idl_string_ (throwing on parse error) and adds the results to the `IdlArray`. If
+  some of the definitions refer to other definitions, those must be present
+  too.  For instance, if _idl_string_ says that `Document` inherits from `Node`,
+  the `Node` interface must also have been provided in some call to `add_idls()`
+  or `add_untested_idls()`.
+
+### `add_untested_idls(idl_string)`
```

Comment:
It may be worth being a little bit more explicit about when this is useful. I'm honestly not entirely clear myself, for example `entries-api/idlharness.html` has:

```
idl_array.add_untested_idls('[Exposed=(Window,Worker)] interface File {};');
```

But in its idl file it has:

```
partial interface File {
    readonly attribute USVString webkitRelativePath;
};
```

Does this mean we're not testing for `webkitRelativePath`? Or that in all **other** cases when one uses a partial interface we're incorrectly testing too much?